Translations/Web/lang_pt.php

```diff
@@ -0,0 +1,69 @@
+<?php // Keka lang - version 0.3 - Started on 26/11/09 - by aone
+
+//
+//
+// Language: Portuguese
+// Translated by BR Lingo (contact@brlingo.com)
```

Comment:
Note that this (the mail) is just visible by me or the server admins, since PHP comments are not rendered in the final html code. Just so you know!